doc/article/en-US/i18n-with-aurelia.md

```diff
@@ -268,31 +268,46 @@ the necessary type definitions (d.ts) for the plugins dependencies.
 Here we show how you can do that for i18next and the i18next-xhr-backend, but this
 should be applicable to every other backend choice.
 
-The way to get hold of those is using [typings](https://github.com/typings/typings).
-You can install the typings for [i18next](http://i18next.com/) using this command:
+The way to get hold of those is using using NPM or perhaps Yarn as with other packages.
+In this case the typinhs are at [@types/i18next](https://www.npmjs.com/package/@types/i18next),
+along with the instructions. The command is
 
 ```Shell
-typings install dt~i18next --save --global
+npm install --save @types/i18next
```

Comment:
I guess ´--save-dev´ would be more appropriate, so that it would be saved under "devDependencies"?

Reply:
Ah, this is true! I read WebPack doesn't care, but it's better nevertheless to do that. Patience guys, and keep th comments coming. :) I'm off to sleep now, but I'll fix tomorrow when there's a better chance of not making mistakes.